docs/getting-started.md

```diff
@@ -15,6 +19,21 @@ You may need to download the [CoreOS Application Signing Key](http://coreos.com/
 
 New releases of `fcct` are backwards compatible with old releases unless otherwise noted.
 
+#### Container
+
+This example uses podman, but docker can also be used. Substitute v0.2.0 with the desired version. Note that the `latest` tag corresponds with `master` and not with the latest release.
+
+```bash
+# Pull the desired version
+podman pull quay.io/coreos/fcct:v0.2.0
+
+# Run fcct using standard in and standard out
+podman run -it --rm fcct:v0.2.0 -pretty -strict < your_config.fcc > transpiled_config.ign
+
+# Run fcct using files. Note that the directory containing the files must be mounted into the container
+podman run --rm -v path_to_config_dir:/srv fcct:v0.2.0 -input /srv/your_config.fcc -output /srv/transpiled_config.ign -pretty -strict
```

Comment:
Hmm, were you able to test this? On Fedora/RHEL, I'd expect SELinux permission issues.

Another approach is to bind-mount the file directly with `:z` so it gets relabeled. So e.g.:

```
podman run --rm -v path_to_fcc_file:/config.fcc:z fcct:v0.2.0 -input /config.fcc > output.ign
```

The advantage of this is that you don't have to bind-mount and relabel a whole directory, which is a common gotcha for a lot of people. E.g. doing `-v /home/bob:/srv:z` will wreck your system.

(We could even default the container to use `-input /config.fcc` to make this easier.)

Reply:
I did test this but on my laptop which doesn't have SELinux enabled.

I don't like the idea of `-input /config.fcc` being default because it's less intuitive to not have the default be stdin/stdout. It's also different than the default when not running it in a container.

Reply:
> I don't like the idea of -input /config.fcc being default because it's less intuitive to not have the default be stdin/stdout

Yeah, makes sense. WDYT about the rest of the proposal though? (I.e. bind-mounting the file directly so podman only relabels that bit.)

Alternatively, we could just add `:z` to what you currently have, but mention that the whole directory will be relabeled if it's not already the right label?

Reply:
I like that, just updated to that.